src/Crypto/WebAuthn/Model/Types.hs

```diff
@@ -760,6 +760,35 @@ data AuthenticatorDataFlags = AuthenticatorDataFlags
 -- and
 -- [requesting](https://www.w3.org/TR/webauthn-2/#dictionary-assertion-options).
 -- The CeremonyKind araument specifies which.
+--
+-- Values of this type are send to the client to
+-- [create](https://w3c.github.io/webappsec-credential-management/#dom-credentialscontainer-create)
+-- and
+-- [get](https://w3c.github.io/webappsec-credential-management/#dom-credentialscontainer-get)
+-- a credential. After they have been sent, they have to be stored awaiting the
+-- response from the client to be passed to the
+-- `Crypto.WebAuthn.Operation.Registration.verifyRegistrationResponse`
+-- and `Crypto.WebAuthn.Operation.Authentication.verifyAuthenticationResponse`
+-- functions. However, WebAuthn's ceremonies do not make use of all fields, and
+-- some fields may be constant in the RP implementation. So, for efficiency
+-- reasons, the RP might choose to only serialize a subset of these options.
+-- The following fields are used by the functions mentioned above:
```

Comment:
After they have been sent, at least these fields have to be stored for later verification of the response, the others aren't currently used.

For `Crypto.WebAuthn.Operation.Registration.verifyRegistrationResponse`:
- ...
- ...

For `Crypto.WebAuthn.Operation.Authentication.verifyAuthenticationResponse`:
- ...
- ...